src/main/scala/treadle/executable/ExecutionEngine.scala

```diff
@@ -475,8 +475,21 @@ object ExecutionEngine {
   def apply(annotationSeq: AnnotationSeq, wallTime: UTC): ExecutionEngine = {
     val timer = new Timer
     val t0 = System.nanoTime()
+    val stageOptions = view[StageOptions](annotationSeq)
 
     val circuit = annotationSeq.collectFirst { case TreadleCircuitStateAnnotation(c)           => c }.get.circuit
+
```

Comment:
This supports simultaneous output to both a file and stdout. Is this desirable? Is it possible to pass arguments to annotations and use the argument to determine the file name with a name of "" meaning write to stdout?

Reply:
That makes some sense but I'd prefer to do this under another PR. I think changing it the way you suggest would be a very slight API change or would increase the difficulty of getting the file name right (as it stands it now automatically uses the appropriate tests_run_dir directory and the circuit name).